.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: "🚀 Feature request"
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Minor: consider using the [Behaviour-Driven Development](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications) templates for Narratives and Scenarios:  .

Reply:
Thank you. I will read up on this.